website/docs/plugins-built-in/jobs-ui/overview.md

```
@@ -0,0 +1,156 @@
+    return { ...JobsPlugin, plugins: opts.plugins };
+  }
+};
+```
+
+### Navigation
+
+Adds a **Jobs** menu item to the sidebar, positioned before "Employees", with the briefcase icon. Only visible when the `FEATURE_JOB` feature flag is enabled.
+
+### Route Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)